src/config.ts

```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'http://blockexplorer-1.testnet.saage.io/transactions/{txHash}',
 	},
+
+	{
+		rpc: 'https://tx-endpoint-test.imversed.com:443',
+		rest: 'https://query-endpoint-test.imversed.com:443',
+		chainId: 'imversed-test-1',
+		chainName: 'imversedtestnet',
+		stakeCurrency: {
+			coinDenom: 'IMV',
+			coinMinimalDenom: 'nimv',
+			coinDecimals: 6,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/imversed.png',
```

Comment:
please upload the imversed tokon logo image(s) to /public/assets/tokens/ and instead title it imv.png, not imversed.png

Reply:
Image uploaded and name of it changed to imv.png